packages/permission-log-controller/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@metamask/permission-log-controller",
+  "version": "5.0.2",
+  "description": "Controller with middleware for logging requests and responses to restricted and permissions-related methods",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/core/tree/main/packages/permission-log-controller#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/core/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/core.git"
+  },
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build:docs": "typedoc",
+    "changelog:validate": "../../scripts/validate-changelog.sh @metamask/permission-log-controller",
+    "publish:preview": "yarn npm publish --tag preview",
+    "test": "jest",
+    "test:clean": "jest --clearCache",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@metamask/base-controller": "^3.2.3",
+    "@metamask/json-rpc-engine": "^7.1.1",
+    "@metamask/utils": "^8.1.0"
+  },
+  "devDependencies": {
+    "@metamask/auto-changelog": "^3.1.0",
```

Comment:
We updated `auto-changelog` recently, so to remove the CI failures, this should be:

``` suggestion
    "@metamask/auto-changelog": "^3.4.2",